### ISSUE TITLE
Can't return camera image in Moto G

### ISSUE THREAD

**Issue by bkawakami** [jkwiecien/EasyImage#61]
Can't return camera image in Moto G

The path is in wrong location...

**Comment by jkwiecien** [comment]
I don't have that device. You'll have to try to debug it yourself.

**Comment by neelkadia** [comment]
Not able to select multiple images in MotoG also.

**Comment by jkwiecien** [comment]
What gallery app? Not all galleries support it. Samsung galery for example
doesn't

pon., 26.12.2016, 08:59 użytkownik Neel Kadia <notifications@github.com>
napisał:

**Comment by neelkadia** [comment]
@jkwiecien Yes Gallery App of Moto G. Multi select is not working in same app but it work on my Nexus 5X.

Let me know if you have any 'help-wanted' issues, I can solve/contribute

**Comment by vixir** [comment]
I'm facing the same problem in my Moto G4-plus. 
When I take a pic with com.google.android.GoogleCamera(google camera app) or com.motorola.cameraone (motorola camera app), image is not loading. While running EasyImage  sample, photos are getting copied to EasyImage Sample folder, but it's not displayed in recyclerview. Instead blank space is displayed. If I take photo with third party app like Youcam, I'm able to view the photos.

For gallery picking from camera folder is not working. With other folders it's working.
After going through stacktracec, I found out I was getting this infamous error:

> using fit() or resize() in Picasso : java.io.IOException: Cannot reset

[Picasso issue 364](https://github.com/square/picasso/issues/364)


This issue is fixed in their Picasso 2.5.3 SNAPSHOT. Next planned version is 3.0.0, don't know how soon it will roll out.
I switched to Glide as of now.
